Jenkinsfile

```diff
@@ -1,5 +1,8 @@
 //"Jenkins Pipeline is a suite of plugins which supports implementing and integrating continuous delivery pipelines into Jenkins. Pipeline provides an extensible set of tools for modeling delivery pipelines "as code" via the Pipeline DSL."
 //More information can be found on the Jenkins Documentation page https://jenkins.io/doc/
+
+@Library('github.com/connexta/cx-pipeline-library@master') _
```

Comment:
Is the `_` at the end there a typo?

Reply:
Nope, an annotation needs to be attached to something so this is a way to do that without having to have an actual field or something else to attach the annotation to. Recommended from the [pipelines library docs](https://jenkins.io/doc/book/pipeline/shared-libraries/)

```
For Shared Libraries which only define Global Variables (vars/), or a Jenkinsfile which only needs a Global Variable, the annotation pattern @Library('my-shared-library') _ may be useful for keeping code concise. In essence, instead of annotating an unnecessary import statement, the symbol _ is annotated.
```

Reply:
According to https://github.com/connexta/cx-pipeline-library it looks like it might be required? I don't remember needing to have the underscore...